ref/src/interpolate.f90

```diff
@@ -0,0 +1,136 @@
+module interpolate
+  implicit none
+
+contains
+
+  subroutine interpolate_allocate(odims, interpFactor, idims)
+  implicit none
```

Comment:
Inconsistent indentation.

Reply:
That's how Pritify does it.  Subroutines are indented below a contains.  There is probably an option to change that.  How strongly do you want this fixed?

Reply:
@christopherwharrop-noaa  I'm making the assumption that you want the contains and subroutine at the same level of indentation.

Reply:
The `implicit none` should be indented two spaced.  And, actually, it could simply be deleted since the entire module has an `implicit none`

Reply:
So, no.  Leave the contains and the subroutine the way they are.